### ISSUE TITLE
NavigationBar component: we can't have 2 NavigationBar one over the other (subMenu of the first one is hidden)

### ISSUE THREAD

**Issue by hegou33** [Sage/carbon#5045]
### Desired behaviour

We want to have 2 `NavigationBar`, one over the other, and be able to see all the `subMenu` on it.

### Current behaviour

The `z-index` is the same for the 2 `NavigationBar`, so the subMenu of the first one is hidden by the second one.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/vigilant-rhodes-69hqfs?file=/src/index.js:557-570

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by samtjo** [comment]
Hi @tempertemper - just thought we'd check with you if it's valid HTML to have two `nav` elements on the page like this, both with `role="navigation"`?

**Comment by tempertemper** [comment]
Totally valid to have multiple `<nav>`s, but they should have unique `aria-labels` that describe their purpose, e.g.

```html
<nav aria-label="Site-wide">
    <ul>
        <li>The first item</li>
        <!-- The rest of the nav -->
    </ul>
</nav>
<nav aria-label="Meet the team">
    <ul>
        <li>A team member</li>
        <!-- The rest of the team -->
    </ul>
</nav>
```

<i>Note: no need to use the word 'Navigation' in the `aria-label` as the screen reader will read this out anyway, so would only be repetitive and add noise, This is the kind of thing that would be read "Site-wide, navigation"; adding navigation would read "Site-wide navigation, navigation".</i>

**Comment by Parsium** [comment]
@hegou33 FE-5148 has now been merged with Carbon [version 109.7.0](https://github.com/Sage/carbon/releases/tag/v109.7.0) 🚀 

This introduces a new `GlobalHeader` component which has a greater z-index than `NavigationBar`. We opt-ed for this approach to avoid surfacing a `z-index` prop (which could introduce support problems), and to match Design System (where `GlobalHeader` and `NavigationBar` are separate components).

Now to resolve this issue, Global Nav will need updating to Carbon v109.7.0 and its code refactored to make use of `GlobalHeader`. I will contact Nebula Squad regarding this. In the meantime, I will close this issue since this requires changes outside of Carbon.
